nemo/collections/asr/models/label_models.py

```diff
@@ -441,13 +440,48 @@ def get_embedding(self, path2audio_file):
         mode = self.training
         self.freeze()
 
-        _, embs = self.forward(input_signal=audio_signal, input_signal_length=audio_signal_len)
+        logits, emb = self.forward(input_signal=audio_signal, input_signal_length=audio_signal_len)
 
         self.train(mode=mode)
         if mode is True:
             self.unfreeze()
         del audio_signal, audio_signal_len
-        return embs
+        return emb, logits
+
+    def get_label(self, path2audio_file):
+        """
+        returns label of path2audio_file from classes the model was trained on. 
+        Args
+            path2audio_file: path to audio wav file
```

Comment:
path to audio wav files

Reply:
This function is only for one file, for multiple files, `batch_inference` function would be used

Reply:
I think it is better for users to be strict on "article" usage. in this case, you can say "path to an audio wav file"